Extension/src/packageManager.ts

```diff
@@ -467,3 +437,33 @@ export class PackageManager {
         }
     }
 }
+
+export function VersionsMatch(pkg: IPackage, info: PlatformInformation): boolean {
```

Comment:
Functions should start with a lower case letter.  I know these were previously wrong.  Can you fix them while you're in here?